react/src/data/wrapper.ts

```diff
@@ -52,12 +51,19 @@ export class FormStateWrapper extends AbstractModelsWrapper<FormModelState, Form
     }
 
     override hookModel = (modelName: string) => {
-        let selectFn = useSelector;
-        return selectFn(this.formSelectors.selectFormState(modelName));
+        let m = useSelector(this.formSelectors.selectFormState(modelName));
+        if(m === undefined || m === null) {
+            throw new Error("model could not be hooked because it was not found: " + modelName);
+        }
+        return m;
     }
 
     getFieldFromModel(fieldName: string, model: FormModelState): FormFieldState<unknown> {
-        return model[fieldName];
+        let fv = model[fieldName];
+        if(fv === undefined || fv === null) {
+            throw new Error(`field could not be found in model state: ${fieldName}, ${JSON.stringify(model)}`)
```

Comment:
We are tending towards errors where names are bound to values: `field=${fieldName} could not be found in model=${JSON.stringify(model)}`